user-guide/how-to-host.md

```diff
@@ -0,0 +1,83 @@
+On all operating systems you have the possibility to open single ports manually. TripleA uses 
+port 3300 by default.
+DO NOT TURN OFF YOUR FIREWALL FOR A LONGER PERIOD OF TIME UNDER ANY CIRCUMSTANCES
+Finding your Firewall-Settings
+Windows
+
+Open your Control Panel, change the View By setting to "Small/Large Icons" and click on "Windows 
+Firewall">Allow a program or feature through Windows Firewall
+Linux
+
+Based on the distribution you are using this will differ. On Debian based systems (including Ubuntu)
+ you can use ufw (uncomplicated firewall) instead of manually messing with iptables. With ufw you just enter sudo ufw allow 3300 in a terminal. Done.
+If you are using Linux you probably know how to figure out how to do this on other distributions. 
+There are just too many to list them all.
+Mac
+
+I have no idea... (If you are a mac user, please tell me so I can edit this in here later)
+Configuring your Router
+
+Assuming yor Computer is not connected directly to the internet, but connected to a router instead 
+which is most likely the case, we need to tell your Route it's supposed to route all TripleA-related 
+traffic to your computer.
+This is called "Port-Forwarding". You can find tons of guides for every router model out there in
+the Internet. It would be impossible to list all of those here!
+Make sure your Router forwards Port 3300 to Port 3300 on your local machine.
+If you are asked for the Protocol select TCP.
+
+
+### Troubleshooting
+
+Computers often don't work like you want them to, here are some tricks that might come in handy 
+if something is not working for you:
+Try temporarily disabling your Firewall. If everything is working fine with your Firewall disabled,
+ you haven't setup your Firewall correctly. If it's still not working your Router isn't correctly 
+ forwarding the traffic to your PC.
+
+[canyouseeme.org](http://canyouseeme.org) is a website that helps you checking if your configuration is 
+working. Just enter the port you want to check (in our case 3300).
+
+If you still have any questions, feel free to ask!
+
+
+### Bot Hosting
+Anyone that can host can host bots.
+
+Now you will need to place a bot folder or multiple ones, in the same TripleA program folder. 
+The below is an example of one of my bots. Using windows. Other operating systems are slightly 
+different.
+
+```
+@echo off
+SET PORT=3363
+SET BOT_NUMBER=25
+
+SET LOBBY_HOST=45.79.144.53
+SET LOBBY_PORT=3304
+
+java -server -Xmx320m -Djava.awt.headless=true -classpath bin/triplea.jar games.strategy.engine.framework.headlessGameServer.HeadlessGameServer -Ptriplea.game.host.console=false -Ptriplea.game= -Ptriplea.server=true -Ptriplea.port=%PORT% -Ptriplea.lobby.host=%LOBBY_HOST% -Ptriplea.lobby.port=%LOBBY_PORT% -Ptriplea.name=Bot%BOT_NUMBER%_Pras1 -Ptriplea.lobby.game.hostedBy=Bot%BOT_NUMBER%_Pras1 -Ptriplea.lobby.game.supportEmail=prastle7@hotmail.com -Ptriplea.lobby.game.comments="automated_host" -Ptriplea.lobby.game.reconnection=172800
```

Comment:
I just realized: We should probably remove @prastle 's email here

Reply:
Even worse than that, the recommended way to run a bot is via the 'run_bot.bat' script, to download that and modify. This is a great example of how we documented things multiple times and lead to documentation drift.

Reply:
I did a quick update of this: e9e061f

To update to be proper instructions of launching the bat file I'd like to leave for another iteration where we overhaul this document even more.

Reply:
Note, used this script: https://github.com/triplea-game/triplea/blob/master/game-headless/scripts/run_bot.bat, for the list of updated params, they have been simplified since this was written. 
Again the proper solution is to give instructions to unzip the bot and then run the included bat script, these steps are redundant... Leaving for another day or person while I'm focusing on Mod toolbox right now.